generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -858,7 +994,10 @@ message GetSelfCalibrateLastDateAndTimeResponse {
 message GetSelfCalibrateLastTemperatureRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
-  int64 self_calibrate_step = 3;
+  oneof self_calibrate_step_enum {
+    SelfCalStep self_calibrate_step = 3;
+    int64 self_calibrate_step_raw = 4;
```

Comment:
@astarche Here is a int 64 self-cal step in RFInstru thats why I was curious

Reply:
Yikes thanks. I don't know why that is. I created AB#1804296 to make sure we sign this off in API review. I'm going to see if I can find the G code that implements these methods and send Nagashree an email.